docs/changehistory/NextVersion.md

```diff
@@ -6,6 +6,14 @@ publish: false
 
 Table of contents:
 
+- [Quantity](#quantity)
+- [Electron 32 support](#electron-32-support)
+
+## Quantity
+
+- The `minWidth` property on FormatProps now works as documented.
+- The `spacer` property on FormatProps now indicates the space used between composite components, it defaults to a single space, and there is no longer a ':' prepended. If a ':' spacer is desired, `spacer` has to be set accordingly. This is to streamline the behavior with the documentation and native APIs.
+- Added support for bearing and azimuth format types (e.g. bearing `N45°30'10"E`). A new phenomenon "Direction" for these will be added to our units library soon, but they work just as well with the angle phenomenon for now. Persistence values for both bearing and azimuth are to be provided counter-clockwise from an east-base (inspired by PowerPlatform).
```

Comment:
should we be mentioning "PowerPlatform"? how much of itwinjs consumers know about that? isnt it an internal concept?

Reply:
It is the reason why we chose this default instead of North-Clockwise. Historically, MicroStation has done that, and much of our data is probably stored in that way. So.... not sure. By not mentioning PowerPlatform we'd be making this choice seem arbitrary. What do you think?

Reply:
I searched for [Power Platform](https://bentleysystems.service-now.com/community?id=kb_article&sysparm_article=KB0099402) and I guess this is good enough of an explanation for it, so im ok leaving this in.